01-js-intro-and-jquery/style.css

```diff
@@ -0,0 +1,12 @@
+* {
```

Comment:
Wouldn't use a ``*`` selector here, since exercise is proposing to center ``section`` elements. I would delete this definition and ad ``text-align: center;`` to your ``section`` selector defined below. :+1: